kustomize/base/quay.deployment.yaml

```diff
@@ -33,6 +33,18 @@ spec:
                   name: cluster-trusted-ca
               - secret:
                   name: extra-ca-certs
+        - name: postgres-certs
+          projected:
+            sources:
+            - secret:
+              name: postgresql-ca
+              optional: true
+            - secret:
+              name: postgresql-client-certs
+              optional: true
```

Comment:
I had to use the following indentation where name and optional appear under secret. I wouldn't copy from here since Github formatting seems to be odd.
```
- name: postgres-certs
  projected:
    sources:
      - secret:
          name: postgresql-ca
          optional: true
      - secret:
          name: postgresql-client-certs
          optional: true
```